backend/src/yeomanui.ts

```diff
@@ -8,7 +8,13 @@ import { ReplayUtils, ReplayState } from "./replayUtils";
 const datauri = require("datauri"); // eslint-disable-line @typescript-eslint/no-var-requires
 const titleize = require('titleize'); // eslint-disable-line @typescript-eslint/no-var-requires
 const humanizeString = require('humanize-string'); // eslint-disable-line @typescript-eslint/no-var-requires
-import * as defaultImage from "./defaultImage";
+import * as defaultImage from "./images/defaultImage";
+import * as errorVSCode from "./images/vscode/errorMessage";
+import * as infoVSCode from "./images/vscode/infoMessage";
+import * as warnVSCode from "./images/vscode/warnMessage";
+import * as errorTheia from "./images/theia/errorMessage";
+import * as infoTheia from "./images/theia/infoMessage";
+import * as warnTheia from "./images/theia/warnMessage";
```

Comment:
All could be in one file.
import { errorMessage, infoMessage, warnMessage  } from "./images/vscode/Messages";

check isInBAS in ./images/vscode/Messages.ts to return the right image.

Reply:
done